libraries/AP_HAL_ChibiOS/hwdef/JFB100/defaults.parm

```diff
@@ -0,0 +1,943 @@
+ACRO_BAL_PITCH,1
```

Comment:
Most of these should not be in here as it depends on what sort of vehicle the thing is attached to!

What defaults *actually* need to be in here?